tools/proteore_topgo/topGO.xml

```diff
@@ -0,0 +1,268 @@
+<tool id="topGO" name="Enrichment analysis" version="2021.04.12">
+    <description>(Human, Mouse, Rat)[topGO]</description>
+    <requirements>
+        <requirement type="package">R</requirement>
+        <requirement type="package">r-ggplot2</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.hs.eg.db</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.mm.eg.db</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.rn.eg.db</requirement>
+        <requirement type="package" version="1.62.0">bioconductor-graph</requirement>
+        <requirement type="package" version="1.46.0">bioconductor-annotationdbi</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-go.db</requirement>
+        <requirement type="package" version="2.36.0">bioconductor-topgo</requirement>
+    </requirements>
+    <stdio>
```

Comment:
this stdio you can remove please use error detection: https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection